packages/babel-generator/src/generators/modules.js

```diff
@@ -53,6 +53,16 @@ export function ExportAllDeclaration(node: Object) {
   this.word("from");
   this.space();
   this.print(node.source, node);
+  if (node.assertions?.length) {
+    this.space();
+    this.word("assert");
+    this.space();
+    this.token("{");
+    this.space();
+    this.printList(node.assertions, node);
+    this.space();
+    this.token("}");
+  }
```

Comment:
Nit: since this is repeated different times, we can extract it into a `printAssertions(node)` function.

Reply:
dd4b991 👍